indico_toolkit/snapshots/snapshot.py

```diff
@@ -186,25 +199,26 @@ def _infer_text_col(self):
         elif "document" in self.df.columns:
             self.text_col = "document"
         else:
-            raise ToolkitInstantiationError(
-                f"You must set 'text_col' from {list(self.df.colmns)}"
-            )
+            error_message = self._format_error_message("text_col")
```

Comment:
should these be _format_column_error_message?

Reply:
Yes, this reminds me that we need testing to ensure exceptions fire off correctly.

Reply:
Fixed